build/package/Microsoft.DotNet.Cli.Nupkg.targets

```diff
@@ -11,46 +11,111 @@
     </PropertyGroup>
     
     <ItemGroup>
-      <ProjectsToPack Include="$(ProjectsSrcDirectory)/Microsoft.DotNet.Cli.Utils" >
+      <MicrosoftDotNetCliUtilsInputs Include="$(ProjectsSrcDirectory)/Microsoft.DotNet.Cli.Utils/**/*.cs;
```

Comment:
I removed this. I would have to hard code the outputs for this to work properly, plus all it does is invoke dotnet build on the project. I will let dotnet build itself figure out if something changed.